docs/faq/README.md

```diff
@@ -123,3 +123,50 @@ ReferenceError: process is not defined
 The reason for this is that Redux ships CommonJS by default, and expects `process.env.NODE_ENV` to be set. In the browser, this is not the case. The solution for this is to use the ESM build of Redux instead. It's importable from <a href="https://unpkg.com/browse/redux@4.0.5/es/redux.mjs">`node_modules/redux/es/redux.mjs`</a>. Note that the `.js` version still expects `process.env.NODE_ENV`, but the `.mjs` version does _not_. You'll want to use the `.mjs` version.
 
 And everything should run in the browser as expected.
+
+## Why do certain styles pierce the shadow DOM?
+
+> "Doesn't shadow DOM provide total encapsulation?"
+
+Nope! As written in the [spec](https://www.w3.org/TR/css-scoping-1/#inheritance):
+
+> The top-level elements of a shadow tree inherit from their host element.
+
+What this means is that [_inheritable_](https://gist.github.com/dcneiner/1137601) styles continue to inherit in shadow DOM, will _pierce_ the shadow DOM and affect your component's styling.
```

Comment:
can we add the most common use cases people will run into here?
something like 

"this includes for example color or font"

Reply:
done